src/inverseRobot.js

```diff
@@ -24,7 +24,21 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const fixedRob = {};
```

Comment:
не скорочуй назви змінних в цьому нічого доброго немає
```suggestion
  const fixedRobot = {};
```